packages/bolt-connection/src/bolt/bolt-protocol-v1.js

```diff
@@ -454,6 +455,24 @@ export default class BoltProtocol {
     return observer
   }
 
+  /**
+   * Send a TELEMETRY through the underlying connection.
+   *
+   * @param {object} param0 Message params
+   * @param {number} param0.api The API called
+   * @param {object} param1 Configuration and callbacks callbacks
```

Comment:
```suggestion
   * @param {object} param1 Configuration and callbacks
```
Isn't is just callbacks, actually?

Reply:
Just for now, but It might change with configurations like `flush` in the future. I'm just give some tip for myself of the future.